### ISSUE TITLE
Research initialised RAM

### ISSUE THREAD

**Issue by Lapixx** [ams-hackers/gbforth#284]
As an extension of #283, we can look into supporting _initialised RAM_ as mentioned in https://www.taygeta.com/forth/dpanse.htm#E.5.

This would either mean that the words `,` (comma) and `c,` (c-comma) are able to "write" into RAM memory space at _compile-time_ (in addition to ROM), or that some other mechanism is available to initialise RAM in an easier way.

**Comment by lf94** [comment]
I don't believe that makes any sense, unless we think of initialised RAM as "delayed" ?
